Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -107,178 +129,195 @@ static void EFB_Write(u32 data, u32 addr)
 	}
 }
 
+u32 dbat_table[1 << (32 - 17)];
+u32 ibat_table[1 << (32 - 17)];
```

Comment:
should we put these anywhere else, or is this okay? maybe 17 should be a #define, much like the TLB parameters, with a comment explaining it?